nixos/modules/services/misc/ooklaserver.nix

```diff
@@ -0,0 +1,201 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.ooklaserver;
+in
+{
+  options.services.ooklaserver =
+    let
+      inherit (lib) mkOption types;
+    in
+    {
+      enable = lib.mkEnableOption "Ookla Speedtest server";
+      package = lib.mkPackageOption pkgs "ooklaserver" { };
+      domain = mkOption {
+        description = "Domain to use.";
+        default = null;
+        type = with types; nullOr nonEmptyStr;
+      };
```

Comment:
```suggestion
      useACMEHost = mkOption {
        description = ''
          Use a certificate generated by the NixOS ACME module for the given host.
          Note that this will not generate a new certificate - you will need to do so with `security.acme.certs`.
        '';
        default = null;
        type = with types; nullOr nonEmptyStr;
      };
      
      tlsCertificate = mkOption {
        description = "TLS certificate to use. Use together with `tlsKey`.";
        default = lib.mkIf (cfg.useACMEHost != null) "${config.security.acme.certs.${cfg.useACMEHost}.directory}/cert.pem";
        defaultText = /* … */;
        type = types.nonEmptyStr;
      };
      
      tlsKey = mkOption {
        description = "TLS private key to use. Use together with `tlsCertificate`.";
        default = lib.mkIf (cfg.useACMEHost != null) "${config.security.acme.certs.${cfg.useACMEHost}.directory}/key.pem";
        defaultText = /* … */;
        type = types.nonEmptyStr;
      };
```